### PR TITLE
COMMERCE-4868 autocomplete component now displays custom views

### PR DESCRIPTION
**Pull request by fabio-d-m**

### CODE REVIEW

**Comment by FabioDiegoMastrorilli** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/test/dev/components/Autocomplete.js [hunk 1]
```diff
@@ -12,21 +12,41 @@
...
-	initialLabel: 'Min',
-	initialValue: 'initial-value',
+	infinityScrollMode: true,
 	inputName: 'test-name',
 	itemsKey: 'productId',
 	itemsLabel: 'externalReferenceCode',
-	onUpdate: (value, itemData) =>
+	onValueUpdated: (value, itemData) =>
 		// eslint-disable-next-line no-console
 		console.log(`Value: ${value}`, `Data: ${JSON.stringify(itemData)}`),
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-frontend-js/test/dev/components/Autocomplete.js:48]: @FabioDiegoMastrorilli do we really need a Console log?

File: modules/apps/commerce/commerce-frontend-js/test/dev/components/Autocomplete.js [hunk 2]
```diff
@@ -39,7 +59,7 @@ launcher('autocomplete-2', 'autocomplete-root-2', {
...
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-frontend-js/test/dev/components/Autocomplete.js:64]: console.log?

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**